runelite-client/src/main/java/net/runelite/client/callback/Hooks.java

```diff
@@ -274,6 +275,13 @@ public static void draw(MainBufferProvider mainBufferProvider, Graphics graphics
 				stretchedGraphics = (Graphics2D) stretchedImage.getGraphics();
 
 				lastStretchedDimensions = stretchedDimensions;
+				
+				/*
+					Fill Canvas before drawing stretched image to prevent artifacts.
+				*/
+				graphics.setColor(Color.BLACK);
+				graphics.fillRect(0, 0, client.getCanvas().getWidth(), client.getCanvas().getHeight());
+				graphics.setColor(Color.WHITE);
```

Comment:
Is this setColor unnecessary?

Reply:
Hmm, looks like it is.

Reply:
The Black setColor is necessary. The white one may or may not be, but I had asked you specifically deathbeam. and you suggested i set it back

Reply:
If you remove it... does it still work?

Reply:
The black setColor? no, it paints the background white when in fixed resized mode (while not taking up the whole screen) the White one? as my configuration stands yes it works fine, but idk exactly how the graphics hand off will affect any other plugins etc, So I set it back just to be sure.